src/main/webapp/less/table-reflow.less

```diff
@@ -0,0 +1,96 @@
+/**
+ * Licensed to Apereo under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work
+ * for additional information regarding copyright ownership.
+ * Apereo licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License.  You may obtain a
+ * copy of the License at the following location:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+/*---------------------------------------------------
+    TABLE REFLOW Mixin
+  ---------------------------------------------------
+    A mixin to make table element with .reflow Class
+    to reflow on mobile
+  ---------------------------------------------------*/
+
+
+.createtable-reflow( @background;  @color; @border-color; @th-width-mobile; @table-reflow-max-width; @min-width; @max-width ) { 
+  /* Create the table reflow */
+    table.reflow {
```

Comment:
ditto, `<table class="reflow">` is not used, is this a mixin added for future use?